pkg/controller/policyrecommendation/policyrecommendation_controller.go

```diff
@@ -288,6 +288,17 @@ func (r *ReconcilePolicyRecommendation) Reconcile(ctx context.Context, request r
 		return reconcile.Result{RequeueAfter: utils.StandardRetry}, nil
 	}
 
+	// Render a Policy Recommendation component for Windows if the cluster has Windows nodes.
+	hasWindowsNodes, err := common.HasWindowsNodes(r.client)
+	if err != nil {
+		r.status.SetDegraded(operatorv1.ResourceNotReady, "OS not found", err, logc)
```

Comment:
Judging by the implementation of HashWindowsNodes, I think it should be a `ResourceReadError` and perhaps the following message would be better.

> Unable to determine if there are windows nodes.